style-editor.scss

```diff
@@ -19,7 +19,7 @@ body {
 	@include media(mobile) {
 		overflow-x: hidden;
 
-		.editor-block-list__layout,
+		:not(.editor-inner-blocks) > .editor-block-list__layout, // Target only the top level layout element, or nested blocks will also be affected.
```

Comment:
I am aware that this rule is relatively exotic, but it's the shortest and most readable way I could write it.

To explain what's going on — any _nested_ block will always have both `editor-block-list__layout` and `editor-inner-blocks` as parent elements. Which means any CSS rules we apply to `editor-block-list__layout` will cascade into nested blocks. 

By writing this rule, `:not(.editor-inner-blocks) > .editor-block-list__layout`, we are targeting any instance of `editor-block-list__layout` that is _not_ a descendant of `editor-inner-blocks`. Which incidentally means, only the top level one is matched. 

I'd love for a simpler rule, but it absolutely has to be solid enough to target only the to level element.

Reply:
Yeah, this unfortunate, but will have to do for now.